### DISCUSSION TITLE
Analysis of APK doesnt finish !

### DISCUSSION THREAD

**Discussion by ramazansancar** [Pithus/bazaar#143] [Q&A] [answered]
It has been about 1 day, but the report has not yet been concluded. Network part is my priority. :/

Report Url: https://beta.pithus.org/report/20945a35b1fdaf41ece6b06181228ffcd11d41f41496ea9900d92ed5d28b7810

**Comment by evilcel3ri** [comment, marked as answer]
Hi! I haven't looked at the prod data yet. But fwiw, running your apk on a local version of Pithus results this for the network tab 
![Screenshot 2023-04-03 at 10-44-42 Pithus report of ParaPlus - com maslak paraplus](https://user-images.githubusercontent.com/3540752/229459038-9db999b9-ccec-4172-932d-e0a3ea260edb.png)

**Reply by ramazansancar** [reply]
Thanks for your answer
